source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -44,11 +44,41 @@ matching data is inserted.
 Find
 ----
 
-The ``find()`` method is called on the ``Collection`` object that
-references the collection you want to query. The method accepts a query
-document that describes the documents you want to retrieve. For more
-information on how to specify your query document, see our guide on
-how to :doc:`Specify a Query </fundamentals/crud/query-document>`.
+You can call the ``find()`` method on a ``Collection`` object. The
+method accepts a query document that describes the documents you want to
+retrieve. For more information on how to specify your query document,
+see our guide on how to :ref:`node-fundamentals-query-document`.
```

Comment:
[question]

Definitely non-blocking, but wanted to ask a question. Are you concerned that the title of the topic might change? Since you don't specify the title here, if the title of the topic changes, it might make this sentence not read grammatically correct.

Reply:
I guess I wasn't thinking about that, I will change so that even if the title changes, the sentence will still read correctly.